TreeAPI.js

```diff
@@ -82,9 +82,9 @@ WikiTreeAPI.Person = class Person {
 
 } // End Person class definition
 
-// To get a Person for a given id, we POST to the API's getPerson action. When we get a result back,
+// To get a Person for a given id, we POST to the APIs getPerson action. When we get a result back,
```

Comment:
The possessive here in the original is, I think, correct. It is the getPerson action/method of the API. Same with jQuery's .ajax call() below.

Reply:
These updates look great, @GeoffRiley; thanks for the effort. I noted one possible change above, though it's not critical. Your edits to Tree.js have a small conflict with a previous merge that need to be resolved before your spelling/grammar fixes can be pulled through.